### ISSUE TITLE
Feature Request: Searchable descriptions

### ISSUE THREAD

**Issue by samerickson** [just3ws/nerd-font-cheatsheets#1]
Hey really great job on the cheat sheets, I have not been able to find any other resources on the subject. That being said, the functionality that is missing for me is the ability to have these is a single file that I can easily open with a vim keybinding, then search for the icon I am looking for. For instance, searching for "battery" would provide all locations of a glyph matching that description.

**Comment by ryanoasis** [comment]
Just stumbled across this again.
@samerickson, this is great work by @just3ws :bowing_man: and maybe this isn't quite what you want but we've overhauled the cheat sheet on the website, you can see all 'battery' related icons: https://www.nerdfonts.com/cheat-sheet?set=battery

If you want something in the markdown that is searchable then it probably isn't what you want. There are css class names that could probably easily be mapped to the markdowns @just3ws has here..

**Comment by samerickson** [comment]
@ryanoasis Yes, I did not mean for it to sound like I am not appreciating the work that has gone into this, in fact this is the best solution I have ran into. Through, my thoughts were is this is intended for usage in the terminal, it would be nice to be able to search through them as well. At the time of me writing this, I was prepping for mid terms, but I have a bit of time before the school year ramps up, I have time to add descriptions. 

The we interface, at least for me, is borderline unusable. It just feels clunky, but at least it works, so I guess it's better than nothing.

**Comment by just3ws** [comment]
Hey @samerickson and @ryanoasis! Thanks, glad you've found it useful. I've been on a break for past few days so haven't had a chance to reply. This set of files is pretty old and was just something I generated over my commute so I could visualize the glyphs I wanted to use in command prompts. The search was nothing more than `ack` and lots of scrolling to find the glyph that was most like what I wanted. 

@samerickson I totally agree about the web interface. The biggest reason I created this was because I could view the glyph in the same context I'd be using it. Usually Vim and Zsh. 

FWIW I don't know if the codes have been altered since this was originally generated. I might look at it again on the train and see if it could be cleaned up a bit. Maybe adding some descriptions would be good but I didn't have an easy way to do that that I can recall.

**Comment by just3ws** [comment]
Hah. I just realized that your comment @samerickson was from February. Sorry this fell though the cracks. No disrespect intended by the ridiculously long delay in replying. I just saw the new messages popup over my break.

**Comment by samerickson** [comment]
Hey @just3ws no worries. I have started adding descriptions to as many icons as I can identify without preforming a google search: https://github.com/samerickson/nerd-font-cheatsheets
